src/main/java/com/lambdatest/jenkins/freestyle/report/ReportBuildAction.java

```diff
@@ -61,8 +63,7 @@ public void generateLambdaTestReport() {
                     if (buildDetailNode.get("name").toString().replaceAll("\"", "").equals(buildName)) {
                         String build_id = buildDetailNode.get("build_id").toString();
 
-                        lambdaTestBuildBrowserUrl = "https://automation.lambdatest.com/logs/?build=" + build_id;
-
+                        lambdaTestBuildBrowserUrl = "https://automation.lambdatest.com/build?&build=" + build_id;
```

Comment:
@Shahnawaz-Sk use host name as a constant key ,for https://automation.lambdatest.com as well